lib/kubernetes-deploy/runner.rb

```diff
@@ -263,14 +263,15 @@ def deploy_resources(resources, prune: false)
       run_kubectl(*command)
     end
 
-    def set_kubectl_context
+    def validate_context
       out, err, st = run_kubectl("config", "get-contexts", "-o", "name", namespaced: false, with_context: false)
       available_contexts = out.split("\n")
       if !st.success?
         raise FatalDeploymentError, err
       elsif !available_contexts.include?(@context)
         raise FatalDeploymentError, "Context #{@context} is not available. Valid contexts: #{available_contexts}"
       end
+      KubernetesDeploy.logger.info("Context #{@context} validated")
```

Comment:
We do not really validate it, we only check that it exists. Let's change the wording to "context foobar found"